### DISCUSSION TITLE
How does the latest version of the gateway support https?

### DISCUSSION THREAD

**Discussion by whatbug** [apache/apisix#8497] [Q&A] [answered]
actually, The SSL certificate has been uploaded, and the SSL option in my configuration file is' true '. When I open the domain name to access, I still cannot access https service.

**Comment by whatbug** [comment]
install the apisix service by helm in k8s.

**Reply by whatbug** [reply]
<img width="926" alt="image" src="https://user-images.githubusercontent.com/32671621/206860416-41c8418f-ce79-4eac-b85c-515a8a3d63b2.png">
.....

**Comment by whatbug** [comment, marked as answer]
Hello, Everyone. The problem is that the control panel temporarily does not support the operation of the new API after the Apisix version V3 is updated. You need to request port 9180 to update the certificate, and then modify the configuration file to support ssl access service. I have to say that community documents are really rubbish.

**Reply by tzssangglass** [reply]
You can submit a PR to APISIX Dashboard to improve the documentation.

BTW, the APISIX Dashboard home page also declares compatible versions of APISIX, ref: https://github.com/apache/apisix-dashboard
